src/coreclr/src/gc/gc.cpp

```diff
@@ -15614,6 +15624,7 @@ int gc_heap::joined_generation_to_condemn (BOOL should_evaluate_elevation,
 
         if (joined_last_gc_before_oom)
         {
+            gc_data_global.gen_to_condemn_reasons.set_condition(gen_joined_last_gc_oom);
```

Comment:
>gen_joined_last_gc_oom [](start = 64, length = 22)

I would prefix these with gen_joined_limit to emphasize the fact this is for hardlimit